Search key at find_program-ing nosetests' binary might be too specific

On the platform I'm trying (`QNX` 6.5 with `pkgsrc`), [cmake-ing the workspace](http://ros.org/rosdoclite/groovy/api/catkin/html/adv_user_guide/underlay.html) fails as follows. Binary of `nosetests` is found as `/usr/pkg/bin/nosetests-2.7`.

```
$ cd catkinws && mkdir build && cmake ../src
  :
-- Using CATKIN_TEST_RESULTS_DIR: /home/rosuser/catkin_ws/build/test_results
-- Found gtest sources under '/home/rosuser/catkin_ws/src/gtest': gtests will be built
CMake Warning at catkin/cmake/test/nosetests.cmake:94 (message):
  nosetests not found, Python tests can not be run (try installing package
  'python-nose')
Call Stack (most recent call first):
  catkin/cmake/all.cmake:134 (include)
  CMakeLists.txt:12 (include)


CMake Error at catkin/cmake/assert.cmake:17 (message):


  Assertion failed: check for file existence, but filename
  (RT_LIBRARY-NOTFOUND) unset.  Message: RT Library

Call Stack (most recent call first):
  catkin/cmake/tools/rt.cmake:42 (assert_file_exists)
  catkin/cmake/all.cmake:134 (include)
  CMakeLists.txt:12 (include)


-- Configuring incomplete, errors occurred!
```

`catkin` 0.5.69

I'll send a patch right away.

--- a/cmake/test/nosetests.cmake
+++ b/cmake/test/nosetests.cmake
@@ -88,7 +88,7 @@ endfunction()
 find_program(NOSETESTS nosetests)
 if(NOT nosetests_path)
   # retry with name including major version number
-  find_program(NOSETESTS nosetests2)
+  find_program(NOSETESTS NAMES nosetests2 nosetests-2)
 endif()
 if(NOT NOSETESTS)
   message(WARNING "nosetests not found, Python tests can not be run (try installing package 'python-nose')")

